assets/js/ui-kit.js

```diff
@@ -60,6 +60,16 @@ var Velocity = require('velocity-animate');
     Velocity(panel, bool ? 'slideUp' : 'slideDown', { duration: 300 });
   }
 
+  checkMedia = function(){
```

Comment:
@elisechant No it's not a replacement for css media queries.

It's just a way to check in JS which media query is currently active without having to maintain break-points in two places (JS and CSS).

I have also had an issue in the past with some browsers returning different viewport sizes in JS than the media-query it's currently using (can't remember specifically). So this uses the media-query as the source of truth.

Would use something like this:
```
      window.onresize = function(event) {
        var mediaSize = checkMedia();
        if ( mediaSize == "mobile" ) {
          // Do mobile stuff
        } else {
          // Do other stuff
        }
      };
```

Ref: http://bricss.net/post/22198838298/easily-checking-in-javascript-if-a-css-media-query